utils/report.c

```diff
@@ -753,7 +758,7 @@ void report_sort_tasks(struct uftrace_data *handle, struct rb_root *name_root,
 	}                                                                                          \
 	FIELD_STRUCT(_id, _name, _func, _header, 10)
 
-#define FIELD_CALL(_id, _name, _field, _func, _header)                                             \
+#define FIELD_UNIT(_id, _name, _field, _func, _header)                                             \
```

Comment:
It's not about 'unit' as there's no such thing here.  What I suggested was 'uint' (yeah, it's confusing) which means 'unsigned int'.

Reply:
I was confused. I updated it.